token-validation-spring-test/src/main/java/no/nav/security/token/support/spring/test/MockLoginController.java

```diff
@@ -60,4 +59,41 @@ public Cookie addCookie(
         }
         return cookie;
     }
+
+    @Unprotected
+    @PostMapping("/cookie")
+    public Cookie addCoaaokie(
+        @RequestParam(value = "issuerId") String issuerId,
+        @RequestParam(value = "audience") String audience,
+        @RequestParam(value = "subject", defaultValue = "12345678910") String subject,
```

Comment:
`audience` and `subject` should prob be removed as they can be provided in the request body (more clean this way)